src/githubHelper.ts

```diff
@@ -1329,6 +1342,11 @@ export class GithubHelper {
       this.gitlabHelper
     );
 
+    if (add_issue_information) {
```

Comment:
Would it be better to put this behind a setting?

Reply:
I would think so